fg/main.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/urfave/cli"
+	"log"
+	"os"
+	"text/template"
+)
+
+type CliFlagInfo struct {
+	PackageName string
+	Flags       []FlagType
+}
+
+type FlagType struct {
+	Name           string `json:"name"`
+	Type           string `json:"type"`
+	Value          bool   `json:"value"`
+	Destination    bool   `json:"dest"`
+	Doctail        string `json:"doctail"`
+	ContextDefault string `json:"context_default"`
+	ContextType    string `json:"context_type"`
+	Parser         string `json:"parser"`
+	ParserCast     string `json:"parser_cast"`
+}
+
+var flagTemplate = `// Code generated by fg; DO NOT EDIT.
```

Comment:
⭐⭐ _(seriously consider)_

This template is fairly long, so I think it'd be a good idea to move it into a file? https://github.com/urfave/cli/pull/769 implements something like that

Reply:
This is now done. Kindly review :)